framework/db/conditions/BetweenCondition.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
```

Comment:
Standard Yii header missing.

Reply:
It will be added during release process automatically